chia/cmds/wallet.py

```diff
@@ -925,8 +924,7 @@ def nft_get_info_cmd(
 
 
 # Keep at bottom.
-# Remove coins subcommand from wallet for now pending further testing
-# wallet_cmd.add_command(coins_cmd)
+wallet_cmd.add_command(coins_cmd)
```

Comment:
This adds back a lot more commands than the ones you just fixed. Are you confident that the other ones are OK?

Reply:
they should be, and work when i checked but ill have someone double check right now.